### PR TITLE
Add a few rules in the operator Makefile.

### PR DESCRIPTION
**Pull request by jingyih**
Add "install", "uninstall", "deploy" and "destroy" rules to the operator Makefile.

### CODE REVIEW

File: operator/Makefile
Comment (justinsb) [inline operator/Makefile:77]: I think these are existing from our internal repo (?), so we probably want to add them as-is and then make them better.

Some ideas for how to make them better:

* Use server-side apply
* Maybe rename install to install-crds? (and same for uninstall)
* If deploy relies on docker-build, maybe we should make it a makefile dependency :-)

But these are nits and I vote to merge and iterate!

**Comment by justinsb** [comment]
Thanks - I think this is pulling these across from our internal repo (RIP!), so I think a merge-and-iterate strategy is the right one here.

/lgtm

(Not sure the lgtm-bot is fully working yet, but worth a shot...)

**Comment by justinsb** [comment]
It may be on now... maybe

/lgtm

File: operator/Makefile
Comment (jingyih) [reply operator/Makefile:77]: Yeah these rules are from internal Makefile which I used often when working on the operator code.

**Comment by jingyih** [comment]
> It may be on now... maybe
> 
> /lgtm

Looks like it is not... Shall we merge a PR after getting one lgtm? Not sure if there is specific guidelines for accepting and merging PRs. In etcd repo we used to get 2 LGTM (ideally from different org), but the situation is different for each repo.

**Review by diviner524 — APPROVED** [review]
LGTM

**Pull request merged**
